Makefile

```diff
@@ -0,0 +1,22 @@
+PWN_HOSTNAME="pwnagotchi"
+PWN_VERSION="master"
+
+all: install image clean
+
+install:
+	curl https://releases.hashicorp.com/packer/1.3.5/packer_1.3.5_linux_amd64.zip -o /tmp/packer.zip
```

Comment:
Just wondering: a debian package exists[1], why not use that?

[1] https://packages.debian.org/buster/packer

Reply:
I need a really specific version for the arm provisioner to work